akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -59,10 +62,12 @@ private[remote] object EnvelopeBuffer {
   val SenderActorRefTagOffset = 16 // Int
   val RecipientActorRefTagOffset = 20 // Int
   val ClassManifestTagOffset = 24 // Int
-  val ActorRefCompressionTableVersionTagOffset = 28 // Int
-  val ClassManifestCompressionTableVersionTagOffset = 32 // Int
+  val ActorRefCompressionTableVersionTagOffset = 28 // Int // TODO handle roll-over and move to Short
+  val ClassManifestCompressionTableVersionTagOffset = 32 // Int // TODO handle roll-over and move to Short
```

Comment:
The reason everything is Int aligned is because you will get 4 bytes from the memory anyway. If you make them a Short then it is likely better to get them as an Int and then make two shorts out of it. I preferred aligned access everywhere over space but this can be reevaluated.